docs/tsg_install.md

```diff
@@ -57,8 +58,17 @@ iscsiadm version 6.2.0.874-7
 [root@node-34622 ~]# docker exec kubelet iscsiadm -V
 iscsiadm version 2.0-874
 ```
-
-To resolve this issue, do not install `open-iscsi / iscsi-initiator-utils` on the host nodes when using the Rancher Container Engine (RKE).
+If output returns iscsiadm version for both commands, then you have to remove iscsi from the node. OpenEBS target will use the iscsi inside the kubelet. 
+To resolve this issue, do not install `open-iscsi / iscsi-initiator-utils` on the host nodes when using the Rancher Container Engine (RKE). Run the following commands to remove iscsi packages from the node if it already installed on your Ubuntu host. You can use the similar commands based on your host OS.
```

Comment:
iSCSI packages from the node it it is already...
You can use similar commands...